sponsor_us.md

```diff
@@ -17,40 +17,47 @@ By supporting us you help to make the world a little better. Your company also
+with credit card and will receive an invoice.
 
-## Financial
+* Platinum tier (~600€): Food for the main event 
+* Gold Tier (~500€): Notebooks for the Attendees 
+* Silver Tier (~400€): Food for the coaches training
+* Small Supporter Tier (250€)
 
-To handle donations we work with
-[Travis Foundation UG](http://foundation.travis-ci.org/), a registered
-not-for-profit company (a “Gemeinnützige Organisation” under German law.) Travis
-Foundation can issue a Donation Receipt for accounting and tax deduction. It
-does *not* issue invoices for donations.
+All our sponsors are listed on our website and on our event page on 
+ClojureBridge.org. They'll receive a shoutout at the start and end of the event
+along with their logos in our presentations. All our sponsors will also have
+the opportunity to offer "swag" (stickers, note books, thumb drives, etc.) at 
+the event.
+
+Additionally, those who sponsored in our top three tiers will also get special
+shoutouts on our social media accounts, along with priority placements of their
+logos.
 
-As a sponsor you can also choose to directly pick up the bill for one of our
-expenses, in which case we let the supplier invoice your company directly. This
-can be a great way to make your contribution more concrete and visible ("thank
-you Foocorp Inc. for the lovely t-shirts!"). Keep in mind that in this case we
-might not be able to predict up front the exact amount invoiced, it could for
-instance depend on the amount of sign-ups.
 
-## Budget
+## Financial
+
+There's two ways your company can choose to support us:
+### Direct Sponsorship
+If you've opted for a direct sponsorship tier, then your transaction will be with
+our suppliers directly. In this case, we will give you the exact
+link/message/order that we need, and you make the transaction with that
+company directly. This means normal business billing, with ability to pay by
+credit card, and get an invoice. However, due to the nature of the process,
+we can't predict the exact amount — it would depend on factors such as the
+the number of sign-ups.
 
-This is an example budget based on previous workshops:
+### Support us via donation
 
-* T-shirts: € 655
-* Catering workshop: € 500
-* Catering installfest: € 360
-* Catering coaches training: € 75
-* Breakfast € 100
-* Afternoon snacks € 100
-* Drinks: € 280
-* Stickers € 90
-* Small costs (tags, postage, pens): € 30
+Alternatively, we also take donations via the
+[Travis Foundation UG](http://foundation.travis-ci.org/), a registered
+not-for-profit company (a “Gemeinnützige Organisation” under German law). This 
+is how we can take the small supporter tier, for donations from 250€. The Travis 
+Foundation can issue a Donation Receipt for accounting and tax deduction. It
+does *not* issue invoices for donations.
 
-Get in touch today at Clojurebridge.Berlin (at) gmail.com.
+Get in touch today at <clojurebridge.berlin@gmail.com>.
```

Comment:
one tiny last change, let's add the twitter handle @clojurebridgede here, like @Malwine suggested. 
After that let's merge it \o/